packages/airnode-node/README.md

```diff
@@ -84,66 +64,23 @@ src/
 │   └── verification/    # Request verification
 ├── handlers/            # "Pipeline" type modules that generally serve as entry points
 ├── providers/           # Provider workers and state
+├── reporting/           # Heartbeat implemenetation
 ├── requests/            # Generic modules applicable to different blockchains
 ├── utils/               # General utility functions
 └── workers/             # Utility function that allow for "forking"
     └── cloud-platforms/ # Implementations for specific cloud vendors
 ```
 
-## Development
+### Local invocation
 
-### config.json
-
-Before you can use or invoke Airnode locally, you must have a valid `config.json` in the `__dev__` folder. You can find
+Before you can use or invoke Airnode locally, you must have a valid `config.json` in the `config` folder. You can find
 the specification in the
 [config.json documentation](https://docs.api3.org/airnode/latest/reference/deployment-files/config-json.html).
 
-For your convenience, example `config.json` and `.env` files are provided in the `__dev__` folder. You can simply copy
-these files and remove the `.example` extension.
-
-### Blockchain node
-
-In order to invoke Airnode, you will need to provide the HTTP URL to an existing blockchain node. For your convenience,
-the following blockchain nodes can be started in development mode using:
-
-```sh
-###########################################
-# Ethereum
-###########################################
-# Start a node at http://127.0.0.1:8545 (separate terminal)
-yarn run dev:eth-node
-
-# Deploy the Airnode protocol contracts
-yarn run dev:eth-deploy
-
-# Create onchain requests that can be processed when invoking Airnode
-yarn run dev:eth-requests
-```
-
-For more information, please refer to the
-[operation README](https://github.com/api3dao/airnode/blob/master/packages/airnode-operation/README.md)
-
-### Dummy web API
-
-A "dummy" web API is also included with the operation package. This exposes a simple
-[Express.js](https://github.com/expressjs/express) server with a few hardcoded endpoints.
-
-This API can be started by running:
-
-```sh
-# Start the API at http://localhost:5000 (separate terminal)
-yarn run dev:api
-```
```

Comment:
This is still needed if you want to for example run E2E tests locally, right? I would keep that in as well.

Reply:
Oh, it was there twice. Fair enough, carry on :slightly_smiling_face: